### DISCUSSION TITLE
No identity issued

### DISCUSSION THREAD

**Discussion by Memoryeva** [HewlettPackard/swarm-learning#39] [Q&A]
run-sn

![image](https://user-images.githubusercontent.com/19405661/127626795-710599d5-23a7-44d6-984e-60930852c92a.png)

how to fix it

**Comment by RadhakrishnaJ** [comment]
No issue with above message. Multiple attempts are made to get identity. In subsequent attempt Identities are created as shown in your screenshot.

**Reply by Memoryeva** [reply]
thanks，i hava other question, about this when i use one host, but run docker container 

my sn-1 host-ip=172.17.0.4,
sl-1: host-ip=172.17.0.6  sl-fs-port=16000
sl-2: host-ip=172.17.0.7  sl-fs-port=17000

![image](https://user-images.githubusercontent.com/19405661/127690351-48ca67f3-e14e-4c6c-8d19-ff46adefe3e3.png)

i'm very sorry, maybe question look like stupid
